### ISSUE TITLE
[Bug]: Heltec T114 rev2 bluetooth pairing FIXED_PIN fails

### ISSUE THREAD

**Issue by pharrup** [meshtastic/firmware#5377]
### Category

BLE

### Hardware

Heltec Mesh Node T114

### Firmware Version

2.5.12.aa184e6

### Description

Only able to pair Heltec T114 rev2 with Bluetooth if using random pin. I am using a headless node so wanted to use a fixed pin, but I am unable to pair unless I use a random pin which I have to obtain via the serial console due to a lack of screen. 

Here is the output I get when trying to pair with a fixed pin:

```INFO | ??:??:?? 264 BLE Connected to *********
INFO | ??:??:?? 264 BLE pair process started with passkey 649 ***
INFO | ??:??:?? 264 BLE passkey pair: match_request=0
INFO | ??:??:?? 271 BLE pair failed
DEBUG | ??:??:?? 271 [Screen] Show standard frames
DEBUG | ??:??:?? 271 [Screen] Show 0 module frames
DEBUG | ??:??:?? 271 [Screen] Total frame count: 103
DEBUG | ??:??:?? 271 [Screen] Added modules. numframes: 0
DEBUG | ??:??:?? 271 [Screen] Finished build frames. numframes: 2
INFO | ??:??:?? 274 BLE Disconnected, reason = 0x13
DEBUG | ??:??:?? 274 PhoneAPI::close()```

CLI output for Bluetooth settings:
```meshtastic --get bluetooth
Connected to radio
bluetooth.enabled: True
bluetooth.fixed_pin: 123456
Completed getting preferences```

2.5.11 and 2.5.9 beta firmware display the same behaviour.

Bluetooth pairing isn't functioning when using a fixed pin and only appears to work when using a random pin.



### Relevant log output

```Shell
INFO | ??:??:?? 264 BLE Connected to *********
INFO | ??:??:?? 264 BLE pair process started with passkey 649 ***
INFO | ??:??:?? 264 BLE passkey pair: match_request=0
INFO | ??:??:?? 271 BLE pair failed
DEBUG | ??:??:?? 271 [Screen] Show standard frames
DEBUG | ??:??:?? 271 [Screen] Show 0 module frames
DEBUG | ??:??:?? 271 [Screen] Total frame count: 103
DEBUG | ??:??:?? 271 [Screen] Added modules. numframes: 0
DEBUG | ??:??:?? 271 [Screen] Finished build frames. numframes: 2
INFO | ??:??:?? 274 BLE Disconnected, reason = 0x13
DEBUG | ??:??:?? 274 PhoneAPI::close()
```

**Comment by garthvh** [comment]
What is your pairing mode set to? It also needs to be fixed pin.  Random is the default if the device has a screen.

**Comment by fifieldt** [comment]
This is related to or a duplicate of: https://github.com/meshtastic/firmware/issues/4565

**Comment by fifieldt** [comment]
TL;DR - we need to set t114 to fixed pin by default since some don't have screens.

**Comment by fifieldt** [comment]
Tracking progress in the earlier bug
